### ISSUE TITLE
Image links throws

### ISSUE THREAD

**Issue by David-Rushton** [David-Rushton/morello.markdown#30]
```shell
# Throws:  Error: The filename, directory name, or volume label syntax is incorrect. : <some-path>
md-cli "![foo](https://ichef.bbci.co.uk/live-experience/cps/624/cpsprodpb/vivo/live/images/2022/8/17/7c0c9a05-8195-4f9f-8d8d-b1b891da3a60.jpg)"
```

**Comment by David-Rushton** [comment]
I suspect we only support images read from the file system.  However as that isn't covered by tests it may be entirely broken.

**Comment by David-Rushton** [comment]
Confirmed.  We support the file system.

**Comment by David-Rushton** [comment]
Fixed in #34
